model-utils/src/test/resources/io/github/ygojson/model/utils/schema/JsonSchemaGeneratorAcceptanceTest.testInlineDataSchemas.VersionInfo.received.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema" : "https://json-schema.org/draft/2019-09/schema",
+  "type" : "object",
+  "properties" : {
+    "version" : {
+      "type" : "string",
+      "description" : "Semantic Versioning of the YGOJSON data and files.\n\nMajor version represents breaking changes on the data model and minor versions non-breaking changes.\nPatch versions are non-breaking data updates.",
```

Comment:
We should test without descriptions because we should be able to change them freely to improve the documentation.

Reply:
This was from a previous implementation/refactor, should be removed